fuse/dl/models/model_multihead.py

```diff
@@ -54,6 +54,12 @@ def __init__(
             raise Exception(
                 "Neither conv_inputs nor backbone_args are None. One must be set (conv_inputs soon to be deprecated)"
             )
+        # we don't use the number of input channels.
+        # it's just kept for backward compatibility
+        if conv_inputs is not None:
+            self.conv_inputs = [inp[0] if isinstance(inp, Tuple) else inp for inp in conv_inputs]
+        if backbone_args is not None:
+            self.backbone_args = [inp[0] if isinstance(inp, Tuple) else inp for inp in backbone_args]
 
         self.conv_inputs = conv_inputs
```

Comment:
@alex-golts,

Here is your issue, you override here the conversion you did in lines 57-62

Reply:
thanks, missed it!
still not sure how come when I ran the tests locally they passed (in the same branch), but never mind I guess :-)